src/vm/amd64/PInvokeStubs.asm

```diff
@@ -11,12 +11,13 @@
 include AsmMacros.inc
 include AsmConstants.inc
 
-PInvokeStubForHostWorker        equ ?PInvokeStubForHostWorker@@YAXKPEAX0@Z
-extern PInvokeStubForHostWorker:proc
-
 extern GenericPInvokeCalliStubWorker:proc
 extern VarargPInvokeStubWorker:proc
 
+ifdef FEATURE_INCLUDE_ALL_INTERFACES
```

Comment:
This looks reasonable to me but we need to make sure the corresponding code in frames.cpp is consistent. perhaps change the code in frames.cpp to be #ifdef WIN64 && FEATURE_INCLUDE_ALL_INTERFACES